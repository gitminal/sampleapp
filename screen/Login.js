import { View, Text, StyleSheet, TextInput, StatusBar, Image, ScrollView, useColorScheme } from 'react-native'
import React from 'react'
import { useState } from "react";
import Colors from '../constant/Colors';
import { TouchableOpacity } from 'react-native-gesture-handler';
import Icon from 'react-native-vector-icons/Ionicons';
import constantElement from '../constant/constantElement';
const Login = ({ navigation }) => {
    const [email, setEmail] = useState('');
    const [chEmail, setchEmail] = useState(true);
    const [emailValidError, setEmailValidError] = useState('');

    const [Password, setPassword] = useState('');
    const [chPassword, setchPassword] = useState(true);
    const [passwordValidError, setpasswordValidError] = useState('');
    const handleValidEmail = () => {
        let reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w\w+)+$/;
        if (email == "" || email == undefined || email == null) {
            setEmailValidError('Email address must be enter');
            setchEmail(false);
            return false;
        } else if (!reg.test(email)) {
            setEmailValidError('Enter valid email address');
            setchEmail(false);
            return false;
        } else {
            setEmailValidError("");
            setchEmail(true);
            return true;
        }
    };
    const handlePassword = () => {
        var reg = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/;
        if (Password == "" || Password == undefined || Password == null) {
            setpasswordValidError('Password  must be enter');
            setchPassword(false);
            return false;
        } else if (!reg.test(Password)) {
            setpasswordValidError('Enter valid Password must contain atleast 8 character, capital & lowercase letter,special & numeric character');
            setchPassword(false);
            return false;
        } else {
            setpasswordValidError("");
            setchPassword(true);
            return true;
        }
    };
    return (
        <ScrollView showsVerticalScrollIndicator={false}>
            <View style={styles.container}>
                <StatusBar barStyle="dark-content" backgroundColor="#ffff" />
                <View style={styles.logintext}>
                    <Text style={styles.textlog}>Login</Text>
                    <Text style={styles.subtext}>Welcome Home My Friend</Text>
                </View>
                <View style={{ marginTop: 12 }}    >
                    <View style={styles.inputView}>
                        <Icon name="mail-outline" size={constantElement.size} style={styles.inputicon} />
                        <TextInput
                            style={styles.TextInput}
                            placeholder="Email"
                            autoCorrect={false}
                            autoCapitalize="none"
                            onChangeText={setEmail}
                            onEndEditing={handleValidEmail}
                            testID="Emailtest"
                        />

                    </View>
                    <View>
                        {chEmail == true ? null : <Text style={{ color: "red" }} testID="error">{emailValidError}</Text>}
                    </View>
                </View>
                <View style={styles.inputView}>
                    <Icon name="lock-closed-outline" size={constantElement.size} style={styles.inputicon} />
                    <TextInput
                        style={styles.TextInput}
                        placeholder="Password"
                        secureTextEntry={true}
                        onChangeText={setPassword}
                        onEndEditing={handlePassword}
                        testID="Passwordtest"

                    />
                </View>
                <View>
                    {chPassword == true ? null : <Text style={{color: Colors.red}}>{passwordValidError}</Text>}
                </View>
                <TouchableOpacity style={styles.loginBtn} onPress={() => {
                    if (handleValidEmail() && handlePassword()) {
                        navigation.replace("Signup")
                    }
                }}>
                    <Text style={styles.loginbtntext} testID="Signuptest">LOG IN</Text>
                </TouchableOpacity>

                <View style={{ marginTop: 20, flexDirection: "row" }}>
                    <View style={styles.remember} />

                    <Text style={styles.textremem}>Remember Pasword</Text>

                </View>
                <Text onPress={() => { navigation.navigate("Forgotpassword") }} testID="Forgotpassword" style={styles.textforgot}>Forgot Pasword ?</Text>
                <Text style={styles.account}>Dont have an account ?</Text>
                <View style={{ flexDirection: "row", alignItems: "center", justifyContent: "center" }}>
                    <Text style={styles.textplease}>Please</Text>
                    <Text onPress={() => { navigation.navigate("Signup") }} testID="Signup" style={styles.textsignup}> SIGN UP</Text>
                </View>
                <View>
                    <View style={{ ...styles.loginfacebook, backgroundColor: Colors.loginfacebookbackground }}>
                        <Image style={styles.image} source={require('../asset/facebook.png')} />
                        <Text style={{ ...styles.loginbtntext, color: Colors.textlogfacebook }}>LOG IN WITH FACEBOOK</Text>
                    </View>
                </View>
                <View>
                    <View style={{ ...styles.loginfacebook, backgroundColor: Colors.loggooglebackground }}>
                        <Image style={styles.image} source={require('../asset/google.png')} />
                        <Text style={{ ...styles.loginbtntext, color: "#ef3038" }}>LOG IN WITH GOOGLE</Text>
                    </View>
                </View>
            </View>
        </ScrollView>
    )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: "#fff",
        padding: 30
    },
    logintext: {
        marginTop: 30
    },
    textlog: {
        fontSize: 30,
        color: "black",
        marginTop: 20,
        fontWeight: "bold"
    },
    subtext: {
        fontSize: 14,
        marginTop: 20
    },
    inputView: {
        marginTop: 20,
        flexDirection: "row"
    },
    TextInput: {
        paddingLeft: 50,
        borderBottomWidth: 0.3,
        flex: 1,
        fontSize: 12
    },
    inputicon: {
        marginTop: 15,
        position: "absolute",
        
    },
    loginBtn:
    {
        borderRadius: 5,
        height: 45,
        alignItems: "center",
        justifyContent: "center",
        marginTop: 30,
        backgroundColor: Colors.primaryColor,
    },
    loginbtntext: {
        color: "white",
        fontSize: 18,
        fontWeight: "bold"
    },
    remember: {

        width: 12,
        height: 12,
        marginTop: 4,
        borderColor: Colors.light,
        borderWidth: 0.5,
    },
    textremem: {
        paddingLeft: 15,
        fontSize: 12
    },
    textforgot: {
        width: 100,
        fontSize: 12,
        marginTop: 10,
    },
    account: {
        marginTop: 20,
        textAlign: 'center',
        color: "#404040",
        fontWeight: "400"
    },
    textplease: {
        color: "#404040",
        fontWeight: "400"
    },
    textsignup: {
        color: Colors.primaryColor,
        fontWeight: "500"
    },
    loginfacebook: {
        borderRadius: 5,
        height: 45,
        alignItems: "center",
        marginTop: 10,
        flexDirection: "row",
    },
    image: {
        alignItems: 'flex-start',
        width: 20,
        height: 20,
        margin: 20

    }
})
export default Login;

