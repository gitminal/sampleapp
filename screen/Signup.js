import { View, Text, StyleSheet, TextInput, StatusBar, ScrollView } from 'react-native'
import React, { useState } from 'react'
import Icon from 'react-native-vector-icons/Ionicons';
import Colors from '../constant/Colors';
import { TouchableOpacity } from 'react-native-gesture-handler';
import constantElement from '../constant/constantElement';

const Signup = ({ navigation }) => {
    const [name, setname] = useState('');
    const [chName, setchName] = useState(true);
    const [nameValidError, setnameValidError] = useState('');

    const [email, setEmail] = useState('');
    const [chEmail, setchEmail] = useState(true);
    const [emailValidError, setEmailValidError] = useState('');

    const [Password, setPassword] = useState('');
    const [chPassword, setchPassword] = useState(true);
    const [passwordValidError, setpasswordValidError] = useState('');

    const [PasswordRetype, setPasswordRetype] = useState('');
    const [chPasswordRetype, setchPasswordRetype] = useState(true);
    const [PasswordRetypeValidError, setPasswordRetypeValidError] = useState('');
    const handleName = () => {
        var reg = /^[a-zA-Z ]{2,40}$/;

        if (name == "" || name == undefined || name == null) {
            setnameValidError('Please Enter Name ');
            setchName(false);
            return false;
        } else if (!reg.test(name)) {
            setnameValidError('Enter Valid Name');
            setchName(false);
            return false;
        } else (name == name)
        {
            setchName(true);
            setnameValidError("");
            return true;
        }
    };

    const handleValidEmail = () => {
        let reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w\w+)+$/;
        if (email == "" || email == undefined || email == null) {
            setEmailValidError('Email Address Must Be Enter');
            setchEmail(false);
            return false;
        } else if (!reg.test(email)) {
            setEmailValidError('Enter Valid Email Address');
            setchEmail(false);
            return false;
        } else {
            setEmailValidError("");
            setchEmail(true);
            return true;
        }
    };
    const handlePassword = () => {
        let reg = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$/;
        if (Password == "" || Password == undefined || Password == null) {
            setpasswordValidError('Password Must Be Enter');
            setchPassword(false);
            return false;
        } else if (!reg.test(Password)) {
            setpasswordValidError('Enter valid Password must contain atleast 8 character , capital & lowercase letter,special & numeric character');
            setchPassword(false);
            return false;
        } else {
            setpasswordValidError("");
            setchPassword(true);
            return true;
        }
    };
    const handlePasswordRetype = () => {
        var repass = PasswordRetype.trim();
        if (repass == "" || repass == undefined || repass == null) {
            setPasswordRetypeValidError(' Enter Password Again');
            setchPasswordRetype(false);
            return false;
        } else if (repass != Password) {
            setPasswordRetypeValidError('Password And Retype Password must be same');
            setchPasswordRetype(false);
            return false;
        } else (repass == Password)
        {
            setchPasswordRetype(true);
            setPasswordRetypeValidError("");
            return true;
        }
    };
    return (
        <ScrollView showsVerticalScrollIndicator={false}>
            <View style={styles.container}>
                <StatusBar style={styles.statusbarstyle}/>
                <View style={styles.logintext}>
                    <Text style={styles.textlog}>SIGN UP</Text>
                    <Text style={styles.subtext}>Welcome Home My Friend</Text>
                </View>
                <View style={{ marginTop: 15 }}>
                    <View style={styles.inputView}>
                        <Icon name="person-outline" size={constantElement.size}  style={styles.inputicon} />
                        <TextInput
                            style={styles.TextInput}
                            placeholder="Name"
                            onChangeText={setname}
                            onEndEditing={handleName}
                        />
                    </View>
                    <View>
                        {chName == true ? null :<Text style={{ color: "red" }}>{nameValidError}</Text>}
                    </View>
                </View>
                <View style={styles.inputView}>
                    <Icon name="mail-outline" size={constantElement.size}  style={styles.inputicon} />
                    <TextInput
                        style={styles.TextInput}
                        placeholder="Email"
                        autoCorrect={false}
                        autoCapitalize="none"
                        onChangeText={setEmail}
                        onEndEditing={handleValidEmail}
                        testID="Email"
                    />
                </View>
                <View>
                    {chEmail == true ? null :<Text style={{ color: "red" }}>{emailValidError}</Text>}
                </View>
                <View style={styles.inputView}>
                    <Icon name="lock-closed-outline" size={constantElement.size}  style={styles.inputicon} />
                    <TextInput
                        style={styles.TextInput}
                        placeholder="Password"
                        secureTextEntry={true}
                        onChangeText={setPassword}
                        onEndEditing={handlePassword}
                        testID="Password"

                    />

                </View>
                <View>
                    {chPassword == true ? null :<Text style={{ color: "red" }}>{passwordValidError}</Text>}
                </View>
                <View style={styles.inputView}>
                    <Icon name="lock-closed-outline" size={constantElement.size}  style={styles.inputicon} />
                    <TextInput
                        style={styles.TextInput}
                        placeholder="Password Retype"
                        secureTextEntry={true}
                        onChangeText={setPasswordRetype}
                        onEndEditing={handlePasswordRetype}
                    />
                </View>
                <View>
                    {chPasswordRetype == true ? null :<Text style={{ color: "red" }}>{PasswordRetypeValidError}</Text>}
                </View>
                <View style={styles.rememberview}>
                    <View style={styles.remember} />

                    <Text style={styles.textremem}>Terms & Conditions</Text>
                </View>
                <TouchableOpacity style={styles.loginBtn} onPress={() => {
                    if (handleName() && handleValidEmail() && handlePassword() && handlePasswordRetype()) {
                        navigation.replace("Login")
                    }
                }}>
                    <Text style={styles.loginbtntext} testID="Signup">SIGN UP</Text>
                </TouchableOpacity>
                <Text style={styles.account}>Already  have an account ?</Text>
                <View style={styles.loginview}>
                    <Text style={styles.textplease}>Please</Text>
                    <Text onPress={() => { navigation.navigate("Login") }} style={styles.textsignup} testID="Signuptest"> Login</Text>
                </View>

            </View>
        </ScrollView>
    )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: "#fff",
        padding: 30
    },
    logintext: {
        marginTop: 30
    },
    textlog: {
        fontSize: 30,
        color: "black",
        marginTop: 20,
        fontWeight: "bold"
    },
    subtext: {
        fontSize: 15,
        marginTop: 20
    },
    inputView: {

        marginTop: 15,
        flexDirection: "row"
    },
    TextInput: {
        paddingLeft: 50,
        borderBottomWidth: 0.5,
        flex: 1,
        fontSize: 12
    },
    inputicon: {
        marginTop: 15,
        position: "absolute",
    },
    loginBtn:
    {
        borderRadius: 5,
        height: 45,
        alignItems: "center",
        justifyContent: "center",
        marginTop: 40,
        backgroundColor: Colors.primaryColor,
    },
    loginbtntext: {
        color: "white",
        fontSize: 20,
        fontWeight: "bold"
    },
    remember: {

        width: 12,
        height: 12,
        marginTop: 4,
        borderColor: Colors.light,
        borderWidth: 0.5,
    },
    textremem: {
        paddingLeft: 10,
        fontSize: 13
    },
    textforgot: {
        marginTop: 12,
        fontSize: 12
    },
    account: {
        marginTop: 20,
        textAlign: 'center',
        color: "#404040"
    },
    textplease: {
        color: "#404040",
        fontWeight: "400"
    },
    textsignup: {
        color: Colors.primaryColor,
        fontWeight: "500"
    },
    loginfacebook: {
        borderRadius: 5,
        height: 50,
        alignItems: "center",
        justifyContent: "space-evenly",
        marginTop: 10,
        backgroundColor: Colors.primaryColor,
        flexDirection: "row",
    },
    image: {
        alignItems: 'flex-start',
        width: 20,
        height: 20,
    },
    loginview:{
    flexDirection: "row", 
    alignItems: "center", 
    justifyContent: "center" 
    },
    rememberview:
    { marginTop: 20, flexDirection: "row", },
    statusbar:
        { marginTop: 20, flexDirection: "row", }
    
})
export default Signup;
