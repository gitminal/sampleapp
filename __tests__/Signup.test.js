import React from "react";
import { render } from "@testing-library/react-native"
import Login from "../screen/Login";
import { shallow, mount } from 'enzyme';
import Adapter from "enzyme-adapter-react-16"
import enzyme from 'enzyme';
import { fireEvent, screen } from "react-native-testing-library";
import renderer from 'react-test-renderer';
import { debug } from "react-native-reanimated";
import { View, Text, TouchableOpacity,styles ,TextInput} from 'react-native';
import Signup from "../screen/Signup";
enzyme.configure({ adapter: new Adapter() });
describe('<Login />', () => {
    it('renders correctly', () => {
        const rendered = renderer.create(<Signup />).toJSON();
        expect(rendered).toBeTruthy();
    });
    it('render default element', () => {
        const { getAllByText, getByPlaceholderText } = render(<Signup />)
        expect(getAllByText("SIGN UP").length).toBe(1);
        expect(getAllByText("Welcome Home My Friend").length).toBe(1);
        expect(getByPlaceholderText("Name"));
        expect(getByPlaceholderText("Email"));
        expect(getByPlaceholderText("Password"));
        expect(getByPlaceholderText("Password Retype"));

    })
    it('show Invalid Username and password', () => {
        const { getByTestId, getByText, queryAllByText, getByPlaceholderText } = render(<Signup />)
        expect(getByPlaceholderText("Name"));
        fireEvent.changeText(getByTestId("Email"));
        fireEvent.changeText(getByTestId("Password"));
        expect(getByPlaceholderText("Password Retype"));
        queryAllByText('Name must be enter');
        queryAllByText('email address must be enter');
        expect(queryAllByText('Password  must be enter').length).toBe(0);
    })
   
    it('go to Login', () => {
        const navigation = { navigate: () => { } }
        spyOn(navigation, 'navigate');
        const page = render(<Signup navigation={navigation} />)
        const Login = page.getByTestId('Signuptest');
        fireEvent.press(Login);
        expect(navigation.navigate).toHaveBeenCalledWith("Login");
    })
//   
    it('validate user inputs, and provides error messages', async () => {
        const { getByTestId, getByText, queryByTestId } = render(<Signup />)
        fireEvent.changeText(getByTestId("Email"), {
            target: { onchangetext: "" },
        });
        fireEvent.changeText(getByTestId("Password"), {
            target: { onchangetext: "" },
        });
        fireEvent.press(queryByTestId("Signup"));
        expect(getByText("Enter email.")).toBeInTheDocument();
    })

//     })
//     const handlePassword = jest.fn();
//     const eventData = {
//         nativeEvent: {
//             email : "",
//         },
//     };
//            let wrapper;
//     wrapper = shallow(<Login />);
//     // mock $(...).modal and localStorage.setItem
//     wrapper.instance().handleValidEmail({ Email: { value: '' } });
//     expect(wrapper.setEmail).toEqual({ Email: '', titleError: "email invalid" });
//     wrapper = shallow(<App />);
//     // mock $(...).modal and localStorage.setItem
//     wrapper.setState({ title: '' });
//     wrapper.instance().onEditNote({ target: { value: '0' } });
//     expect(wrapper.state.titleError).toEqual({ body: '', bodyError: "password invalid" });

//     const handleValidEmail = jest.fn();
//     let reg = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w\w+)+$/;

// const Email = 'Jon@jon.com';

// const { getByPlaceholderText } = render(
//   <View>
//     <TextInput placeholder="Email" onChangeText={handleValidEmail} />
//   </View>
// );

// fireEvent.changeText(getByPlaceholderText('Email'), Email);
    
//       it('increment counter correctlry', () => {
//         const setEmail = jest.fn();
//         const useStateSpy = jest.spyOn(React, 'useState');
//         useStateSpy.mockImplementation((init) => [init, setEmail]);
//          expect(setEmail).toHaveBeenCalledWith(1);
//     })
//     it('go to signup1', () => {
//         const instanceOf = renderer.create(<Login />).getInstance();
//       const   handleValidEmail =('jon@jon.com');
//         expect(handleValidEmail).toEqual('jon@jon.com')
//     });

//     // it("should validate form fields", async () => {
//     //     const mockhandleValidEmail = jest.fn();
//     //     render(<Login handleValidEmail={mockhandleValidEmail} />);
//     //     fireEvent.input(screen.getByTestId("Emailtest"), {
//     //       target: {
//     //         value:"Email"
//     //       }
//     //     });

//     //     fireEvent.input(screen.getByRole("Passwordtest", { Password: "Passwordtest" }), {
//     //       target: { value:"password" }
//     //     });

//     //     fireEvent.submit(screen.getByRole("button", { name: Email}));
//     //     expect(await screen.findAllByRole("alert")).toHaveLength(3);
//     //     expect(mockSave).not.toBeCalled();
//     //   });
});