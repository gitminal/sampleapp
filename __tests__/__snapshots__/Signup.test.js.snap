// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders correctly 1`] = `
<RCTScrollView
  showsVerticalScrollIndicator={false}
>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#fff",
          "flex": 1,
          "padding": 30,
        }
      }
    >
      <View
        style={
          Object {
            "marginTop": 30,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "black",
              "fontSize": 30,
              "fontWeight": "bold",
              "marginTop": 20,
            }
          }
        >
          SIGN UP
        </Text>
        <Text
          style={
            Object {
              "fontSize": 15,
              "marginTop": 20,
            }
          }
        >
          Welcome Home My Friend
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 15,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "marginTop": 15,
            }
          }
        >
          <Icon
            name="person-outline"
            size={18}
            style={
              Object {
                "marginTop": 15,
                "position": "absolute",
              }
            }
          />
          <TextInput
            onChangeText={[Function]}
            onEndEditing={[Function]}
            placeholder="Name"
            style={
              Object {
                "borderBottomWidth": 0.5,
                "flex": 1,
                "fontSize": 12,
                "paddingLeft": 50,
              }
            }
          />
        </View>
        <View />
      </View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 15,
          }
        }
      >
        <Icon
          name="mail-outline"
          size={18}
          style={
            Object {
              "marginTop": 15,
              "position": "absolute",
            }
          }
        />
        <TextInput
          autoCapitalize="none"
          autoCorrect={false}
          onChangeText={[Function]}
          onEndEditing={[Function]}
          placeholder="Email"
          style={
            Object {
              "borderBottomWidth": 0.5,
              "flex": 1,
              "fontSize": 12,
              "paddingLeft": 50,
            }
          }
        />
      </View>
      <View />
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 15,
          }
        }
      >
        <Icon
          name="lock-closed-outline"
          size={18}
          style={
            Object {
              "marginTop": 15,
              "position": "absolute",
            }
          }
        />
        <TextInput
          onChangeText={[Function]}
          onEndEditing={[Function]}
          placeholder="Password"
          secureTextEntry={true}
          style={
            Object {
              "borderBottomWidth": 0.5,
              "flex": 1,
              "fontSize": 12,
              "paddingLeft": 50,
            }
          }
        />
      </View>
      <View />
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 15,
          }
        }
      >
        <Icon
          name="lock-closed-outline"
          size={18}
          style={
            Object {
              "marginTop": 15,
              "position": "absolute",
            }
          }
        />
        <TextInput
          onChangeText={[Function]}
          onEndEditing={[Function]}
          placeholder="Password Retype"
          secureTextEntry={true}
          style={
            Object {
              "borderBottomWidth": 0.5,
              "flex": 1,
              "fontSize": 12,
              "paddingLeft": 50,
            }
          }
        />
      </View>
      <View />
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 20,
          }
        }
      >
        <View
          style={
            Object {
              "borderColor": "#D5D8DC ",
              "borderWidth": 0.5,
              "height": 12,
              "marginTop": 4,
              "width": 12,
            }
          }
        />
        <Text
          style={
            Object {
              "fontSize": 13,
              "paddingLeft": 10,
            }
          }
        >
          Terms & Conditions
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "#404040",
            "marginTop": 20,
            "textAlign": "center",
          }
        }
      >
        Already  have an account ?
      </Text>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#404040",
              "fontWeight": "400",
            }
          }
        >
          Please
        </Text>
        <Text
          onPress={[Function]}
          style={
            Object {
              "color": "#7F00FF",
              "fontWeight": "500",
            }
          }
        >
           Login
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
